### DISCUSSION TITLE
Requesting to join team

### DISCUSSION THREAD

**Discussion by igniteinm** [ROM-builders/temporary#21580] [General]
My successful builds
https://cirrus-ci.com/build/5541905262444544
https://cirrus-ci.com/build/5704393740255232

Source code to build my ROM
https://github.com/ignitein/

My github username
https://github.com/igniteinm
Local manifest 
https://github.com/igniteinm/local_manifest
Telegram username @igniteinm

**Comment by Apon77** [comment]
Please check telegram
